test/componentprocessor/componentprocessor.test.js

```diff
@@ -82,7 +82,7 @@ describe.only("component parsing support", () => {
                 expect(parsedExtendedComp.children).not.toBeUndefined();
                 expect(parsedExtendedComp.children.length).toBeGreaterThan(0);
 
-                let expectedChildOrder = ["label_a", "label_b"];
+                let expectedChildOrder = ["label_b"];
```

Comment:
Removed `label_a` from test because we no longer append children at parsing time